lib/plop-pack/skel/component/README.md.hbs

```diff
@@ -16,7 +16,7 @@ npm install -S @{{{pkg 'name'}}}/{{{dashCase name}}}
 Then use it in your code as follows:
 
 ```js
-import React, { createElement as h } from 'react';
+import React from 'react';
```

Comment:
Most likely the user will be working within our ecosystem in which case they need to define `h`.

Maybe we should have:
```js
import { createElement as h } from 'react'; // or perhaps: import React from 'react';
```

Reply:
Reverted